app/lib/scorecard/backend.dart

```diff
@@ -96,7 +97,8 @@ class ScoreCardBackend {
     bool onlyCurrent = false,
   }) async {
     if (packageVersion == null || packageVersion == 'latest') {
-      packageVersion = await packageBackend.getLatestVersion(packageName);
+      packageVersion = await taskBackend.latestFinishedVersion(packageName) ??
+          await packageBackend.getLatestVersion(packageName);
```

Comment:
Let's also update the documentation, it was never obvious what `latest` does, or that it was an allowed value.

Is `getScoreCardData` only used by search?